File: Multi_Language_CodeExamples/dart/list_operations.dart
void main() {
  List<int> numbers = [1, 2, 3, 4, 5];
  
  List<int> doubled = numbers.map((n) => n * 2).toList();
  print('Doubled: $doubled');
  
  int sum = numbers.reduce((a, b) => a + b);
  print('Sum: $sum');
}

File: Multi_Language_CodeExamples/csharp/ListOperations.cs
using System;
using System.Linq;
using System.Collections.Generic;

class ListOperations {
    static void Main() {
        List<int> numbers = new List<int> { 1, 2, 3, 4, 5 };
        
        var doubled = numbers.Select(n => n * 2).ToList();
        Console.WriteLine("Doubled: " + string.Join(", ", doubled));
        
        int sum = numbers.Sum();
        Console.WriteLine("Sum: " + sum);
    }
}
